internal/discord/handlers.go

```diff
@@ -0,0 +1,251 @@
+package discord
+
+import (
+	"fmt"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+var (
+	// I am too dumb and tired to figure out how to set permission restrictions for the roles without these vars.. sorry..
+	kickPermission    int64 = discordgo.PermissionKickMembers
+	mutePermission    int64 = discordgo.PermissionModerateMembers
+	banPermission     int64 = discordgo.PermissionBanMembers
+	channelPermission int64 = discordgo.PermissionManageChannels
+	messagePermission int64 = discordgo.PermissionManageMessages
```

Comment:
These vars are meant for limiting who can see/use the functions instead of bot permissions setup

Reply:
ah i see. honestly i would take the route to check to see if they have management/admin role rather than limit visibility

Reply:
if you do end up keeping this block, let's add a comment explaining what this block is

Reply:
From a brief google search, it seems that I can't limit visibility and usage of the command based on role on this end (at least with discordgo). From what I understand, it can only be done in server settings > integrations > command > role & member overrides, so I believe in our case this would just prevent a random person from using them before the command permissions are set up in the server itself

Reply:
ah so this seems to be more of a mod thing @clairefleuret for visibility